src/promptflow/promptflow/azure/operations/_flow_operations.py

```diff
@@ -247,35 +249,46 @@ def _create_remote_flow_via_file_share_path(self, flow_name, flow_type, flow_def
         )
         return rest_flow_result
 
-    def _create_or_update(self, flow, **kwargs):
-        # upload to file share
-        self._resolve_arm_id_or_upload_dependencies(flow)
-
-        rest_flow = flow._to_rest_object()
-
-        # create flow draft
-        rest_flow_result = self._service_caller.create_flow(
-            subscription_id=self._operation_scope.subscription_id,
-            resource_group_name=self._operation_scope.resource_group_name,
-            workspace_name=self._operation_scope.workspace_name,
-            body=rest_flow,
-        )
-
-        return rest_flow_result
-
     def _get(self, flow_id):
         # TODO: support load remote flow with meta
         raise NotImplementedError("Not implemented yet")
 
-    def _list(self, **kwargs):
+    @monitor_operation(activity_name="pfazure.flows.list", activity_type=ActivityType.PUBLICAPI)
+    def list(
+        self,
+        max_results: int = MAX_LIST_CLI_RESULTS,
+        owned_only: bool = True,
```

Comment:
this is not align with cli logic

Reply:
Seems azure.ai.ml sdk also have different interface between sdk and cli for components/jobs list feature. I guess it's because it's not convenient to type `ListViewType` in cli.